website/src/pages/index.tsx

```diff
@@ -1,32 +1,84 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import clsx from 'clsx';
 import Layout from '@theme/Layout';
 import Link from '@docusaurus/Link';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import Texty from 'rc-texty';
-import 'rc-texty/assets/index.css';
 import styles from './index.module.scss';
+import * as icons from "../components/Icons"
+
+const community = [
+  {
+    'icon':'Github',
+    'star': '5.3 K',
+    'title': 'Github',
+    'link': 'https://github.com/datafuselabs/databend'
+  },
+  {
+    'icon':'Slack',
+    'title': 'Join us slack chanel',
+    'link': 'https://link.databend.rs/join-slack'
+  },
+  {
+    'icon':'Twitter',
+    'title': 'Follow our twitter',
+    'link': 'https://twitter.com/Datafuse_Labs'
+  },
+  {
+    'icon':'Youtube',
+    'title': 'Youtube',
+    'link': 'https://www.youtube.com/@databend8089'
```

Comment:
**Idea**: We really need a better channel name.

cc @wubx & @PsiACE, `@databend8089` is so weird.

Reply:
We will change to new in another PR.